src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+
+    <p className="Person__partner">{Partner(person)}</p>
+  </section>
+);
+
+function Partner(person) {
+  const sex = person.sex === 'f' ? 'husband' : 'wife';
+
+  if (person.isMarried) {
+    return `${person.partnerName} is my ${sex}`;
+  }
+
+  return 'I am not married';
+}
```

Comment:
it is redundant to make separate function. just make ternary.

```suggestion
export const Person = ({ person }) => {
  const personPartner = person.sex === 'f' ? 'husband' : 'wife';

  const partner =  person.isMarried 
    ?  ${person.partnerName} is my ${personParnner}`
    : 'I am not married';
(
  <section className="Person">
    <h2 className="Person__name">
      {`My name is ${person.name}`}
    </h2>

    {person.age && (
      <p className="Person__age">{`I am ${person.age}`}</p>
    )}

    <p className="Person__partner">{partner}</p>
  </section>
);
```